srv/modules/runners/cephprocesses.py

```diff
@@ -60,11 +69,18 @@ def _status(search, roles):
 
     status = {}
     local = salt.client.LocalClient()
+
+    ignore_roles = [ 'admin', 'master' ]
+
+    for ig_role in ignore_roles:
+      if roles.has_key(ig_role) and roles:
```

Comment:
probably reversed ie, if `roles and ..` ? if roles doesn't exist, roles.has_key's behaviour would be undefined